docs/get-started/start-teku.md

```diff
@@ -125,6 +125,24 @@ Content-Length: 0
   </TabItem>
 </Tabs>
 
+## Exit Codes
+
+**Exit Code 1**
+
+This code indicates a scenario where Teku has exited with a fatal error, however restarting Teku without changes is a logical step in correcting it.
+
+Example: If the beacon chain controller has issues starting P2P services on a particular port, the system will exit with a fatal error code of 1. This could correct itself with a restart.
+
+**Exit Code 2**
+
+This code indicates a scenario where Teku has exited with a fatal error, but will not be corrected by itself if Teku is restarted.
+
+Example: Within the migrate database command, if an invalid database version has been specified, Teku will exit with a fatal error code of 2. This error will not correct itself after a restart unless changes have been made to the set up.
+
+**Usages**
```

Comment:
I think it would be useful to give a "real-world" example of how these status code can be useful. We could add an example of a systemd.service file for Teku, using things like `RestartPreventExitStatus=` and `RestartForceExitStatus=` to control the behaviour of restarting Teku automatically (or not) in case of failure.

Reference: https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html